### PR TITLE
[Enhancement] New OSF meeting front page

### PR DESCRIPTION
**Pull request by haoyuchen1992**
## Purpose
This PR is to implement new design for OSF meeting front page (Responsive Web Page). All the content has been discussed with community and it's good to go now!

Resolve JIRA  -- [OSF-4696]

## Changes
It doesn't touch any backend or javascript code and only changes content and appearance by using HTML/CSS.   
### Before Change:
screenshot 2015-10-20 14 02 44

### After Change:
screenshot 2015-10-20 14 03 00
screenshot 2015-10-20 14 03 12

## Side Effects
Since it only changes the css and html in OSF meeting landing page, it's almost impossible to have side effect to other pages.

### CODE REVIEW

File: website/static/css/pages/meeting-landing-page.css [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+.blue-icon {
+    color: #337ab7;
+}
+
+.osf-meeting-p {
+    font-size: 16px;
+}
+
+.m-v-40 {
```
Comment (caneruguz) [inline website/static/css/pages/meeting-landing-page.css]: Adding these classes should not be necessary. Why does the margin have to be 40 but not 50 anduse m-v-lg?

File: website/static/css/pages/meeting-landing-page.css [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+.osf-meeting-header .fa-users {
+    border-radius: 85px;
+    box-shadow: 0px 0px 20px #E0E7EC;
+    width: 140px;
+    height: 140px;
+    padding-top: 23px;
+}
+
+.osf-meeting-header .fa-cloud-upload {
```
Comment (caneruguz) [inline website/static/css/pages/meeting-landing-page.css]: Why are these two different classes, most of their rules are very similar.

File: website/static/css/pages/meeting-landing-page.css [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+    border-radius: 208px;
+    box-shadow: 0px 0px 20px #E0E7EC;
+    width: 140px;
+    height: 140px;
+    padding: 14px;
```
Comment (caneruguz) [inline website/static/css/pages/meeting-landing-page.css]: Define padding and margin with helper classes from the style guide like p-sm.

File: website/static/css/pages/meeting-landing-page.css [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+    font-size: 50px;
+}
+
+.osf-meeting-header .fa-custom-8x {
+    font-size: 100px;
+}
+
+.osf-meeting-header .fa-users {
+    border-radius: 85px;
+    box-shadow: 0px 0px 20px #E0E7EC;
```
Comment (caneruguz) [inline website/static/css/pages/meeting-landing-page.css]: I'm not a fan of the shadows, and we don't use it in this way anywhere else. Can you use this instead ```border: 3px solid #E0E7EC;```. See the difference here: 

screen shot 2015-10-21 at 8 46 27 am

File: website/static/css/pages/meeting-landing-page.css [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+    background: url("/static/img/meeting-page/network-web.png") repeat;
+    background-size: contain;
+    height: 1000px;
+    margin-bottom: -1000px;
+    opacity: 0.35;
+    -ms-filter: "progid:DXImageTransform.Microsoft.Alpha(Opacity=25)";
+}
+
+.grey-background {
+    background-color: white;
```
Comment (caneruguz) [inline website/static/css/pages/meeting-landing-page.css]: Use #FFFFFF for consistency.

**Comment by caneruguz** [comment]
Finished CR1, good work on this @haoyuchen1992 .

File: website/static/css/pages/meeting-landing-page.css (hunk 4 above)
Comment (haoyuchen1992) [inline website/static/css/pages/meeting-landing-page.css]: @caneruguz  Following the feedback, I change all of them,  except  this one. I got what you concern that shadow will catch too much unnecessary attention. Therefore, I choose to use smaller shadow. The reasons why I think it is OK to use shadow here:
1. Shadow on the blue background is beautiful (Similar as moon in the morning)
2. This is the front page(marketing page) and is OK to tolerant some inconsistence. 

screenshot 2015-10-21 09 54 22

**Comment by caneruguz** [comment]
@haoyuchen1992 , thank you, the changes are satisfactory. I'll pass it with the shadows around the top section icons but a reminder:

Sense of beauty is difficult to personalize and has lead to a lot of the dispcrepencies in the past in the osf look. We are using a style guide to provide a unifying look. It looks like we may need to add something about use of borders in this way to the style guide. In the meantime try to use existing elements and styles in future work.

**Comment by haoyuchen1992** [comment]
@caneruguz Consistence matters. Thank you, Caner! :)

**Comment by felliott** [comment]
Finished CR2.  @haoyuchen1992, this looks fantastic!  I found a few small issues, but I'm happy for this to move on to 'Ready to Merge' after they are addressed.

**Comment by haoyuchen1992** [comment]
@felliott All are good feedbacks!  Ready for another Review Thanks :)!

**Comment by haoyuchen1992** [comment]
@felliott  As for the alt text, I think it is OK to use abbreviation. Because there are clickable URL for it also. In the logo image, we also don't add the full name of organizations.

**Comment by caneruguz** [comment]
@haoyuchen1992 I don't think it's okay to use abbreviations in the alt text. Alt text is also for screen readers and it's important for accesibility. 

We use logos without full text because there is visual familiarity and it's not as important for view. However is it's only read to a person through a screen reader than it should both include full name and also that it's a logo. So the suggestion from @felliott is valid and you should implement it.

**Comment by haoyuchen1992** [comment]
@caneruguz Make sense! And also it doesn't hurt to add more text. Thanks for the clarification!

**Comment by sloria** [comment]
The share button is currently hidden. Is this intentional?

osf___meetings

**Comment by sloria** [comment]
@haoyuchen1992 Once the missing icon is resolved, this is good to merge.

**Comment by haoyuchen1992** [comment]
@sloria Hi Steve, which browser you are using? I can't replicate the issue. Or can you test it again? Thanks!

**Comment by sloria** [comment]
Ah, gee, apparently uBlock will undiscerningly hide "share" icons. Looks good.

**Pull request merged**